.github/pull_request_template.md

```diff
@@ -0,0 +1,20 @@
+Add / Remove / Edit {entry name} to/from the "{section name}" section.
+
+<details>
+  <summary>Short pitch</summary>
```

Comment:
We could be a little more precise here, so that no misunderstandings (See #32 ) arise: Why do you find this tool awesome? How do you use this tool? What advantages does it have over XYZ?

Reply:
Will do!